lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -16,6 +16,8 @@ setupRemoteFromTarget = (target, httpRequestType, form = null) ->
   options =
     httpRequestType: httpRequestType
     httpUrl: httpUrl
+    forceScrollReset: TurboGraft.getTGAttribute(target, 'force-scroll-reset')
+    forceHeadAssetInsertion: TurboGraft.getTGAttribute(target, 'force-head-asset-insertion')
```

Comment:
Yah, something higher level.  `tg_remote_navigation: "post"`, or something.

Reading the [tg-remote docs](https://github.com/Shopify/turbograft#data-tg-remote), I _think_ there's an implicit "this is for updating a form" message in there.  I think it's fair to make a distinction between in-place updates and submit-and-navigate actions.